anchor/network/src/discovery.rs

```diff
@@ -417,8 +417,7 @@ impl Discovery {
                             "Grouped subnet discovery query yielded no results.",
                         );
                     }
-                    Ok(r) => {
-                        let results = r.into_iter().map(|enr| (enr, None)).collect();
+                    Ok(results) => {
```

Comment:
The Lighthouse code returns an Option<Instant> here. When I first coded it, I ignored it because its purpose was not clear. I'll take a look at it.